embedded_gems/pathogen/app/components/pathogen/tabs/lazy_panel.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if render_eager? %>
+  <%= helpers.turbo_frame_tag(@frame_id) { content } %>
+<% else %>
+  <%= helpers.turbo_frame_tag(
+    @frame_id,
+    src: @src_path,
+    loading: :lazy,
+    refresh: @refresh,
+  ) { render "pathogen/tabs/loading_skeleton" } %>
```

Comment:
Suggestion (blocking): Can you use a different skeleton that doesn't have any text, otherwise I suspect they will complain

Reply:
Fixed in 5cfa229c2cff99cc4ab9ba5937a42fd6553e160a